rollup.config.js

```diff
@@ -3,6 +3,8 @@ import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 import livereload from 'rollup-plugin-livereload';
 import { terser } from 'rollup-plugin-terser';
+import replace from 'rollup-plugin-replace';
+import { env_vars } from './config.js';
```

Comment:
as I understand right, we don't need env vars here, if we use config file.
you passed dotenv to config.js, so envs will be available.

Reply:
if we need envs in this config, so, you can use dotenv here, pass vars directly to replace plugin and then set them in config.js from process.env without using dotenv